examples/svelte-ssr/index.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+require('svelte/register')
```

Comment:
examples must use `import` / `export` syntax

Reply:
Svelte has some problems with export esm `import 'svelte/register'` or `import 'svelte/compiler'` throw "module not found".
Haven’t come up with a solution yet.

Reply:
gonna try it locally and see if it works

Reply:
it works if you specify a file extension, e.g.:

```js
import 'svelte/register.js'
```